testing/upload_validation.ts

```diff
@@ -318,7 +318,7 @@ const baseAuthenticationValidators = {
   endpointDomain: z.string().optional(),
   // The items are technically a discriminated union type but that union currently only has one member.
   postSetup: z.array(setEndpointPostSetupValidator).optional(),
-  networkDomain: z.string().optional(),
+  networkDomain: z.union([z.string().nonempty(), z.string().nonempty().array().nonempty()]).optional(),
```

Comment:
I thought we usually do `z.array(z.string().nonempty()).nonempty()`. but as long as it works.

Reply:
Done